pages/2021/python.md

```diff
@@ -41,3 +26,58 @@ Tuesday, June 22 at 15 UTC
  * Lists
  * Conditionals
  * Functions
+
+## Setup
+
+To follow along with these tutorials, you can either install Python on
+your own computer, or access a Binder link that will create an environment
+automatically.  
+
+### Getting set up on your own computer
+
+If you would like to follow along with the tutorial on your own computer,
+please follow these instructions.
+
+1. Please follow these instructions for
+   [installing Anaconda](https://docs.anaconda.com/anaconda/install/).
+   When this is done, you should have Anaconda Navigator installed.
+2. Create a folder called `python-intro` on your computer.
+3. Download [python-novice-inflammation-data.zip](https://swcarpentry.github.io/python-novice-inflammation/data/python-novice-inflammation-data.zip),
+   and move the file to `python-intro`.
+4. Unzip `python-novice-inflammation-data.zip` into the `python-intro`
+   folder.
+5. [Open Anaconda Navigator](https://docs.anaconda.com/anaconda/user-guide/getting-started/#open-navigator).
+5. Under the "Home" tab on Anaconda Navigator, search for and install
+   "Jupyter Notebook".
+6. Under the "Environments" tab on Anaconda Navigator, click on the
+   arrow for the `base` environment.  Select "Open terminal".
+7. Type `pip install numpy matplotlib` to install the necessary packages
+   and press enter.   
+8. Click on the arrow for the `base` environment again. Select the
+   option to "Open with Jupyter Notebook".
+9. Navigate to the `python-intro` directory, and enter the `data`
+   sub-directory.
+10. Under the "New" button (probably near the upper right corner),
+    select "Python 3" to open a notebook.   
+11. Test the installing by entering
+   ```Python
+   import numpy
+   import matplotlib
+   data = numpy.loadtxt(fname='inflammation-01.csv', delimiter=',')
+   matplotlib.pyplot.plot(data)
+   matplotlib.pyplot.show() 
+   ``` 
+   Press shift-enter to run the cell.  If you get a fancy plot, you're
+   all set!  If you run into any errors, please ask on the 
+   [Discord channel for the Hack Week](https://discord.gg/HdsZkp9M35)
+   in the `#tutorial-python` channel.
```

Comment:
```suggestion
11. Test the installing by entering

        import numpy
        import matplotlib
        data = numpy.loadtxt(fname='inflammation-01.csv', delimiter=',')
        matplotlib.pyplot.plot(data)
        plt.show()

    Press shift-enter to run the cell.  If you get a fancy plot, you're
    all set!  If you run into any errors, please ask on the 
    [Discord channel for the Hack Week](https://discord.gg/HdsZkp9M35)
    in the `#tutorial-python` channel.
```

I have no clue why, but the code block is not rendering correctly.  This will get it to work.